multiqc/utils/config_defaults.yaml

```diff
@@ -169,6 +169,7 @@ module_order:
     - 'rseqc'
     - 'busco'
     - 'goleft_indexcov'
+    - 'disambiguate'
```

Comment:
Ah, apologies - I assumed that the tool ran the alignments as well on the FastQ files. This is fine where it is then..